index.html

```diff
@@ -230,21 +243,23 @@ <h2 class="about-card-title">Skills</h2>
                         <li><a href="#">Terminal</a></li>
                     </ul>
                 </div>
-            </div>
-        </article>
+            </article>
+        </div>
     </section>
 
     <section id="contact-frm" class="c-posts">
-        <article class="c-posts__item pb-0">
+        <article class="c-posts__item pb-0 c-form">
             <h1 class="section-title mt-0">
                 I'm always interested in hearing about new projects, so if you'd like
                 to chat please get in touch.
             </h1>
             <form method="post" action="https://formspree.io/f/xpzewgrz">
```

Comment:
- [ ] As I mentioned above, this section also lacks the margin horizontally at the left and right sides. and at bigger screen sizes, some of the form section even gets out of the display which is not a good user experience. kindly consider fixing it because of ensuring good user experience and to stick with the figma design.
- [ ] Also kindly consider placing the background image in this form section as seen in the figma design.
your design
![zxc](https://user-images.githubusercontent.com/96683843/214653136-0fd003c6-091c-4734-a2fe-573610e954e9.jpg)

figma design
![zxcv](https://user-images.githubusercontent.com/96683843/214653073-805ef0af-fd45-4848-8756-a0d4c5e30023.jpg)

Reply:
@Zuhaib042 those are all individual multiple image with different size, does not know how to fit those as background